NineMensMorris/src/singleplayergame.cpp

```diff
@@ -25,50 +26,223 @@ void SinglePlayerGame::scanSpaces() {
     }
 }
 
+// Populates vector of computer piece indices
+void SinglePlayerGame::getAvailablePieceIndices(){
+    unsigned int i;
+    availableSelectIndices.clear();
+    if (computerColorWhite){
+        for (i = 0; i < availableSelect.size(); i++){
+            availableSelectIndices.push_back(getSpaceIndex(whitePieces[availableSelect[i]]->getSpace()));
+        }
+    } else {
+        for (i = 0; i < availableSelect.size(); i++){
+            availableSelectIndices.push_back(getSpaceIndex(blackPieces[availableSelect[i]]->getSpace()));
+        }
+    }
+}
+
+// solving the redundant move bug
+void SinglePlayerGame::millChecker(){
```

Comment:
since this method isn't being used, it should most likely be removed

Reply:
James has specific plans for it but I agree we should not be pushing large blocks of commented code into our production code.

Reply:
Alright, it's been removed